index.html

```diff
@@ -4264,6 +4264,48 @@ <h3>Spectrum of Privacy</h3>
         </p>
       </section>
 
+      <section class="informative">
+        <h3>Software Trust Boundaries</h3>
+
+        <p>
+There are a variety of trust boundaries that exist in the
+<a href="#ecosystem-overview">ecosystem described by this specification</a>.
+Just like an individual using a web browser trusts the web browser not to betray
+their trust by uploading their personal information to a data broker, a similar
+trust exists between the roles in the ecosystem described by this specification
+and the software that operates on the behalf of each of those roles. For
+example,
+        </p>
+
+        <ul>
+          <li>
+An <a>issuer</a> trusts the issuer software it uses to not issue <a>verifiable
```

Comment:
```suggestion
An <a>issuer</a> trusts the issuer software it uses not to issue <a>verifiable
```

(Sounds better for my non-native English ears. If I am wrong, just ignore this.)

Reply:
Fixed in @TallTed's suggestion.